### PR TITLE
Bump eslint to 3.6.1 & eslint-plugin-import to 1.16.0

### PR DESCRIPTION
**Pull request by wfleming**
cc @codeclimate/review

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,1109 +0,0 @@
-{
-  "name": "codeclimate-eslint",
-  "version": "0.0.3",
-  "dependencies": {
-    "acorn": {
-      "version": "3.3.0",
```
Comment (maxjacobson) [inline npm-shrinkwrap.json]: I think this is too many lines deleted. I think it means you didn't `npm install` before `npm shrinkwrap`

Comment (wfleming) [inline npm-shrinkwrap.json]: Hmmm. Yeah, I don't know what happened there. I just used our `make shrinkwrap` rule. I'll look into it, thanks for noticing.

**Comment by wfleming** [comment]
@maxjacobson Updated. I'll open a separate PR with some potential changes to the `shrinkwrap` make rule based on what I ran into.

**Comment by maxjacobson** [comment]
Sweet, LGTM

**Pull request merged**